src/vorta/utils.py

```diff
@@ -318,3 +320,35 @@ def is_system_tray_available():
         is_available = tray.isSystemTrayAvailable()
 
     return is_available
+
+
+def validate_passwords(firstPass, secondPass):
+    passEqual = firstPass == secondPass
+    passLong = len(firstPass) > 8
+
+    if not passLong and not passEqual:
+        return trans_late('utils', "Passwords must be identical and greater than 8 characters long.")
+    if not passEqual:
+        return trans_late('utils', "Passwords must be identical.")
+    if not passLong:
+        return trans_late('utils', "Passwords must be greater than 8 characters long.")
+
+    return ""
+
+
+def password_transparency(encryption):
```

Comment:
This is good.

Reply:
Maybe there could be a clearer name for this function? Like `display_password_backend()`

Reply:
Yeah, that seems more accurate.